ios/StatusIm/Info.plist

```diff
@@ -1,91 +1,97 @@
-	<array>
-		<string>armv7</string>
-	</array>
-	<key>UISupportedInterfaceOrientations</key>
-	<array>
-		<string>UIInterfaceOrientationPortrait</string>
-	</array>
-	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<false/>
-</dict>
-</plist>
+  <dict>
+    <key>CFBundleDevelopmentRegion</key>
+    <string>en</string>
+    <key>CFBundleDisplayName</key>
+    <string>Status</string>
+    <key>CFBundleExecutable</key>
+    <string>$(EXECUTABLE_NAME)</string>
+    <key>CFBundleIdentifier</key>
+    <string>im.status.ethereum</string>
+    <key>CFBundleInfoDictionaryVersion</key>
+    <string>6.0</string>
+    <key>CFBundleName</key>
+    <string>$(PRODUCT_NAME)</string>
+    <key>CFBundlePackageType</key>
+    <string>APPL</string>
+    <key>CFBundleShortVersionString</key>
+    <string>1.0</string>
+    <key>CFBundleSignature</key>
+    <string>????</string>
+    <key>CFBundleVersion</key>
+    <string>1</string>
+    <key>LSApplicationQueriesSchemes</key>
+    <array>
+      <string>whatsapp</string>
+    </array>
+    <key>LSRequiresIPhoneOS</key>
+    <true/>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+      <key>NSAllowsArbitraryLoads</key>
+      <true/>
+      <key>NSExceptionDomains</key>
+      <dict>
+        <key>api.status.im</key>
+        <dict>
+          <key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
+          <true/>
+        </dict>
+        <key>localhost</key>
+        <dict>
+          <key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
+          <true/>
+        </dict>
+      </dict>
+    </dict>
+    <key>NSCameraUsageDescription</key>
+    <string>We need to access your camera</string>
+    <key>NSContactsUsageDescription</key>
+    <string>We need to access your contacts</string>
+    <key>NSLocationWhenInUseUsageDescription</key>
+    <string/>
+    <key>NSMicrophoneUsageDescription</key>
+    <string>Need microphone access for Instabug and Audio Messages</string>
+    <key>NSPhotoLibraryUsageDescription</key>
+    <string>We need to access your photo storage to give you an ability to select photos</string>
+    <key>UIAppFonts</key>
+    <array>
+      <string>RobotoMono-Medium.ttf</string>
+      <string>SF-UI-Text-Semibold.otf</string>
+      <string>SF-UI-Text-Bold.otf</string>
+      <string>SF-UI-Text-Regular.otf</string>
+      <string>SF-UI-Text-Medium.otf</string>
+      <string>SF-UI-Text-Light.otf</string>
+      <string>SF-UI-Display-Medium.otf</string>
+      <string>SF-UI-Display-Regular.otf</string>
+      <string>SF-UI-Display-Semibold.otf</string>
+      <string>SF-UI-Display-Thin.otf</string>
+      <string>Entypo.ttf</string>
+      <string>EvilIcons.ttf</string>
+      <string>FontAwesome.ttf</string>
+      <string>Foundation.ttf</string>
+      <string>Ionicons.ttf</string>
+      <string>MaterialIcons.ttf</string>
+      <string>Octicons.ttf</string>
+      <string>Zocial.ttf</string>
+      <string>MaterialCommunityIcons.ttf</string>
```

Comment:
why do we need to add new fonts?

Reply:
These were autogenerated - I believe as a leftover from when I targeted API 25. Removed and seems to build fine.